app.js

```diff
@@ -139,6 +176,29 @@ nextMarker = argv.marker || null;
 AWS.config.sslEnabled = argv.sslEnabled;
 var s3 = new AWS.S3();
 
+if (argv.listFile) {
+  listFile = new LineStream();
+  
+  if (argv.listFile === '-') {
+    process.stdin.pipe(listFile);
+  } else {
+    fs.createReadStream(argv.listFile, {flags: 'r'}).pipe(listFile);
+  }
+  
+  // Read list of objects to copy from a file
+  listFile.on('data', function (line) {
+    var newObject = {
+      Key: line.toString(),
+      StorageClass: 'STANDARD',
+      Size: 0
+    };
+    if (checkKeyMatch(newObject)) {
+      objectsToCopy.push(newObject);
+    }
+    startRequests();
```

Comment:
if there is a listFile provided, we will spin up `startRequests()` for that list file in this block, then outside of the if block, we are spinning up a separate `startRequest()`, do we need to call this twice if we are executing this script with the list file parameter?